lib/tasks/groups.rake

```diff
@@ -0,0 +1,53 @@
+namespace :db do
+
+  desc 'Create groups for assignments'
+  task :groups => :environment do
+    puts 'Assign Groups/Students for Assignments'
+    students = Student.all
+    Assignment.all.each do |assignment|
+      15.times do |time|
+        student = students[time]
+        if assignment.short_identifier == 'A1' || assignment.short_identifier == 'A3'
+          group = Group.create(
+              group_name: "#{ student.user_name }" + ' ' + "#{ assignment.short_identifier }",
```

Comment:
I don't think there should be a trailing comma here. Oh, and I'd probably get rid of the use of + ' ' + here, since you can just put it all into one string. Same down on line 23